packages/core/content-type-builder/admin/src/components/FormModalSubHeader/index.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { Typography } from '@strapi/design-system/Typography';
+import upperFirst from 'lodash/upperFirst';
+import { getTrad } from '../../utils';
+import getModalTitleSubHeader from './getModalTitleSubHeader';
+
+const FormModalSubHeader = ({
+  actionType,
+  modalType,
+  forTarget,
+  kind,
+  step,
+  attributeType,
+  attributeName,
+  customField,
+}) => {
+  const { formatMessage } = useIntl();
+
+  if (customField) {
```

Comment:
I think it would be better to create a specific modal type to decouple the code, you'll then be able to update the getModalTitleSuHeader util with your specific case.

Reply:
ok the modalType for customField exists, we can add a new case in the util and use it to get the id.

I had suggested this approach since you had recommended adding instead of modifying. But in that case maybe we should revert this component / condition and just have a condition for the type, something like:

```javascript
...
          type: modalType === 'customField' ? upperFirst(formatMessage(intlLabel)) : upperFirst(
            formatMessage({
              id: getTrad(`attribute.${attributeType}`),
            })
          ),
...
```

Reply:
I'll merge and make the update